FirebaseStorageSwift/Sources/Result.swift

```diff
@@ -91,9 +91,9 @@ public extension StorageReference {
   func list(maxResults: Int64,
             pageToken: String,
             completion: @escaping (Result<StorageListResult, Error>) -> Void) {
-    list(maxResults: maxResults,
-         pageToken: pageToken,
-         completion: getResultCallback(completion: completion))
+    __list(withMaxResults: maxResults,
```

Comment:
list implementation for Swift clients that will find the error handling in Result.swift